pkg/objects/swift.go

```diff
@@ -57,16 +62,23 @@ type SwiftLocation struct {
 	FileExists map[string]bool `yaml:"-"`
 }
 
-func (s SwiftLocation) cacheKey() string {
-	return strings.Join([]string{
+func (s SwiftLocation) cacheKey(name string) string {
```

Comment:
Why is `cacheKey` taking the `name` argument here? If I understand correctly, this breaks the entire purpose of the function, since now it cannot reuse connections across `SwiftLocation` instances anymore.

Reply:
in case when you need a debug, you don't care about the efficiency, but want to see what happens and why. without the name parameter you won't be able to identify where are the source client logs and where  the target client logs.